docs/tutorials/studio-tutorial.md

```diff
@@ -572,6 +572,15 @@ This panel lets you define tests for the services you write by specifying test d
 
 ## Create a Service Test 
 
+### Create connections test data
+Click the `generate` button to automatically generate test data if possible. Otherwise, please manually create test data.
+#### Data Type -- Relational
+Add relational data tables based on databases to which the service's mapping connects. It's unnecessary to provide all columns with test data. 
```

Comment:
Hmm should we make this a bit more user centric, what do you think
maybe something along what/when/how should it be used?
what - A type of embedded data used to specify test data for relational data sources
when - To be used when user is sourcing data from relational db's
how - ...